src/index.html

```diff
@@ -1,13 +1,71 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style/index.css">
+  </head>
+  <body class="body">
+    <div class="container">
+      <div class="card" data-qa="card">
+        <div class="card__picture">
+          <img
+            src="images/imac.jpeg"
+            alt="Buy it!"
+            class="card__img"
+          >
+        </div>
+
+        <div class="card__product-name">
+          <h1 class="card__h1">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+        </div>
+
+        <div class="card__product-id">
+          <p class="card__p">Код товара: 195434</p>
+        </div>
```

Comment:
```suggestion
        <div class="card__product-id">
          <p class="card__product-id--code">Код товара: 195434</p>
        </div>
```
клас у параграфа дивно названий як на мене краще булоб указати наприклад

Reply:
Agree with @erikkaraman that you need to fix classes. But card__product-id--code isn't the best class name too (strange when you have a class with a modifier for the element that doesn’t have a class with bem block or element). Here nice class can be card__product-code